pkg/didcomm/transport/http/outbound.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+//go:generate ../../../../scripts/openssl_env.sh ../../../../scripts/generate_test_keys.sh
```

Comment:
Why not put this under the testdata directory too?

Reply:
sure, might as well since they're used for http transport..